src/lib/wpcli-versions.ts

```diff
@@ -54,7 +54,12 @@ async function getLatestWPCliVersion() {
 }
 
 export const getWPCliVersionFromInstallation = async () => {
-	const { stdout } = await executeWPCli( '.', [ '--version' ] );
+	/**
+	 * This command is used to check the version of WP-CLI
+	 * which doesn't require mounting the project directory
+	 * so we are not passing the projectPath.
+	 */
+	const { stdout } = await executeWPCli( undefined, [ '--version' ] );
```

Comment:
We used to mount '.' here which can take a long time depending on the directory.
For example, locally it mounts the Studio project folder and this takes 5-6s on my machine. 
In CI it takes +20s.

Because we don't need to mount any directories to perform a version check I added a way to skip mounting. 
Passing explicitly `undefined` and documenting its behavior seemed like the cleanest way of doing it.

Reply:
Nice catch 👍